web/src/components/user/UserCLIAndAPITab.vue

```diff
@@ -60,7 +60,7 @@ const usageWithCurl = computed(() => {
   return usage;
 });
 
-const usageWithCli = `# woodpecker setup --server ${address}`;
+const usageWithCli = `# woodpecker-cli setup --server ${address}`;
```

Comment:
Isn't the cli name `woodpecker`?

```suggestion
const usageWithCli = `# woodpecker setup --server ${address}`;
```